src/osm_configurator/control/aggregation_controller.py

```diff
@@ -17,13 +18,13 @@ def __init__(self, model: IApplication):
         Args:
             model (application_interface.IApplication): The interface which is used to communicate with the model.
         """
-        pass
+        self._aggregation_configurator: AggregationConfiguration = model.get_active_project().get_config_manager().get_aggregation_configuration()
```

Comment:
don't get the aggregation configuration in the init. Now if the active project changes, this controller would still access teh old project.